scripts/upgrade

```diff
@@ -27,7 +27,7 @@ final_path=$(ynh_app_setting_get $app final_path)
 if [ "$is_public" = "Yes" ]; then
 	ynh_app_setting_set $app is_public 1
 	is_public=1
-elif [ "$is_public" = "No" ]; then
+else
```

Comment:
If you change this line like that, each upgrade will set is_public to 0. Because, if is_public is equal to 1 (not to Yes) it will use the else statement and set is_public to 0.